### PR TITLE
semgrep 1.10.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
  semgrep:
    * Formula refers to a GitHub issue or pull request that is closed: https://github.com/pypa/setuptools/issues/2355
```

File: Formula/semgrep.rb [hunk 1]
```diff
@@ -188,7 +188,6 @@ def install
 
       # Officially suggested workaround for breaking change in setuptools v50.0.0
       # See: https://sourceforge.net/p/ruamel-yaml/tickets/356/
-      # Relevant Issue: https://github.com/pypa/setuptools/issues/2355
```
Comment (chenrui333) [inline Formula/semgrep.rb:176]: Any code refs for such `brew audit` check failure?

Comment (p-linnane) [reply Formula/semgrep.rb:176]: https://github.com/Homebrew/brew/pull/14326

**Review by p-linnane — APPROVED** [review]

**Comment by spencerdrak** [comment]
@chenrui333 and @p-linnane  thanks for opening this up! We will have to take a look at the formula to avoid the brew audit failures in the future. CC: @emjin

File: Formula/semgrep.rb (hunk 1 above)
Comment (brendongo) [reply Formula/semgrep.rb:176]: How should we referenced closed issues that have relevant discussions?

**Pull request closed without merging**